src/griddyn/comms/commManager.cpp

```diff
@@ -35,7 +35,8 @@ commManager::commManager (const commManager &cm)
     }
 }
 
-commManager::commManager (commManager &&) = default;
+commManager::commManager (commManager &&) noexcept = default;
```

Comment:
Move constructors generally need to be noexcept to be selected. I didn't actually check to see if any of the things I marked could throw, though

Reply:
need to check whether the header marked it as noexcept.  I think the =default will mark it as noexcept

Reply:
https://stackoverflow.com/a/18654089 looks like you're right. I wonder why it warned me then